lib/stax/mixin/dynamodb/local.rb

```diff
@@ -17,9 +17,9 @@ class DynamoDB < SubCommand
 
       no_commands do
 
-        ## client for dynamodb-local endpoint
-        def client
-          @_client ||= ::Aws::DynamoDB::Client.new(endpoint: 'http://localhost:8000')
+        ## client for dynamodb-local endpoint on given port
+        def client(port)
+          @_client ||= ::Aws::DynamoDB::Client.new(endpoint: "http://localhost:#{port}")
```

Comment:
@buccolo Memoization like this of a method with an arg is an anti-pattern: it will lead to potentially surprising behavior if `local-create` and `local-delete` are called in the same process. This is most often not the case, however, so I'm fine to merge this and add an issue to make the memoization robust :-)

Thanks.